VAMobile/env/env.sh

```diff
@@ -50,6 +50,10 @@ else
 fi
 # set demo mode password
 echo "DEMO_PASSWORD=${DEMO_PASSWORD}" >> .env
+
+# set website URLs
+echo "LINK_URL_SCHEDULE_APPOINTMENTS=https://${WEBSITE_PREFIX}va.gov/health-care/schedule-view-va-appointments" >> .env
+
```

Comment:
What is the functional purpose of this that i'm missing?

Reply:
`LINK_URL_SCHEDULE_APPOINTMENTS` was hard-coded as https://www.va.gov/health-care/schedule-view-va-appointments/ in `constants.env`, so even on a staging build, it would take you to the prod site because it was static. This change makes the link dynamic, so it'll adjust the URL based on the environment, which allows us to test in staging. Future links that we want to use SSO with will need to be added to the `env.sh` script similarly